cordova-plugin-one/src/android/ThunderheadProvider.java

```diff
@@ -0,0 +1,57 @@
+package com.thunderhead;
+
+import android.util.Log;
+import android.content.ContentProvider;
+import com.thunderhead.android.api.codeless.OneCodelessInteractionTrackingConfiguration;
+import android.content.ContentValues;
+import android.database.Cursor;
+import android.net.Uri;
+
+public class ThunderheadProvider extends ContentProvider {
+    protected static final String LOG_TAG = "ThunderheadProvider";
+
+    public ThunderheadProvider() {
+    }
+
+    @Override
```

Comment:
I'm assuming all these methods need to be overridden? (beside onCreate where the magic happens)

Reply:
That’s correct